README.md

```diff
@@ -32,6 +32,12 @@ console.log(one) // [1]
 
 function swap(array, first_index, second_index) {
   // -- YOUR ANSWER HERE --
+  if (array.length > 1) {
```

Comment:
What about if the indices that were passed in are greater than the length of the array?